commit 2cb5df8a563ce0aa126929e9b6cb67caafac425f
Merge: b6221df cfb5c24
Author: Scott Lahteine <thinkyhead@users.noreply.github.com>
Date:   Mon Oct 10 13:30:47 2016 -0500

    Merge pull request #4984 from esenapaj/Small-fix-for-PR-#4980
    
    Small fix for the PR #4980 (LIN_ADVANCE bug fix and optimization)

diff --git a/Marlin/stepper.cpp b/Marlin/stepper.cpp
index c394c8a..e3167a1 100644
--- a/Marlin/stepper.cpp
+++ b/Marlin/stepper.cpp
@@ -604,7 +604,7 @@ void Stepper::isr() {
     #endif // ADVANCE or LIN_ADVANCE
 
     #if ENABLED(ADVANCE) || ENABLED(LIN_ADVANCE)
-      eISR_Rate = (timer >> 3) * step_loops / abs(e_steps[TOOL_E_INDEX]); //>> 3 is divide by 8. Reason: Timer 0 runs at 16/8=2MHz, Timer 1 at 16/64=0.25MHz. ==> 2/0.25=8.
+      eISR_Rate = (timer >> 3) * step_loops / abs(e_steps[TOOL_E_INDEX]); //>> 3 is divide by 8. Reason: Timer 1 runs at 16/8=2MHz, Timer 0 at 16/64=0.25MHz. ==> 2/0.25=8.
     #endif
   }
   else if (step_events_completed > (uint32_t)current_block->decelerate_after) {
